samples/web/send-from-aspnetcore-webapi/Core_7.2/WebApplication/WebApplication.Core.csproj

```diff
@@ -6,5 +6,11 @@
   <ItemGroup>
     <ProjectReference Include="..\Shared\Shared.Core.csproj" />
     <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="NServiceBus.Extensions.Hosting" Version="1.0.0-*" />
   </ItemGroup>
+
+  <ItemGroup Label="Force transitive dependencies to use the older version since NServiceBus.Extensions.Hosting brings version 3.0.0">
+    <PackageReference Include="Microsoft.Extensions.Hosting" Version="2.1.1" />
```

Comment:
shouldn't this specify 3.0 instead?

Reply:
There's no 3.0 on the build server, as we've discussed, and the build is failing. I was trying to force the transitive dependency to work with a lower version.

Reply:
since this project is targeting netcoreapp2.1 there should be no issue with the build server because of that.

Reply:
Our package, `Microsoft.Extensions.Hosting` was pulling in 3.0.0 version of `Microsoft.Extensions.Hosting`. Which creates a conflict and the [build is failing](https://builds.particular.net/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=466041&_focus=283).